### PR TITLE
Logging Update (if GPS is not getting a signal)

### PR DESCRIPTION
**Pull request by lanador**
- Updated logging procedure to not depend on GPS
- Added milliseconds as a main timestamp
- Successfully tested - logging of data continued with the loss of GPS signal

### CODE REVIEW

File: HAB/HAB.ino [hunk 1]
```diff
@@ -39,7 +39,7 @@ void setup() {
 
 void loop() {
   GPS::GPSData gpsData = GPS::process();
-  if (gpsData.isValid) {
+
```
Comment (vitosamson) [inline HAB/HAB.ino:42]: Can you fix the indentation on the following lines? They were indented when there was the `if` statement, but that was removed.

File: HAB/HAB.ino [hunk 2]
```diff
@@ -49,7 +49,9 @@ void loop() {
     Sensors::PressureData pressure = Sensors::getPressureData();
 
     Logging::MissionData _data;
-    _data.setGpsData(gpsData);
+    if(gpsData.isValid) {
+      _data.setGpsData(gpsData);
```
Comment (vitosamson) [inline HAB/HAB.ino]: Let's move this line into the existing `if (gpsData.isValid)` statement at line 64.

**Review by vitosamson — CHANGES_REQUESTED** [review]
Looks great, just a couple small nitpicks. Mostly formatting and organization.

File: HAB/HAB.ino (hunk 1 above)
Comment (lanador) [reply HAB/HAB.ino:42]: Done!

File: HAB/HAB.ino (hunk 2 above)
Comment (lanador) [reply HAB/HAB.ino]: Done, please double check if I did it correctly! Thank you!

**Review by vitosamson — APPROVED** [review]

**Pull request merged**